samples/IdentityOIDCWebApplicationSample/Areas/Identity/Extensions/IdentityServiceExtensions.cs

```diff
@@ -13,29 +13,24 @@
 
 namespace Microsoft.AspNetCore.Identity.Service.Extensions
 {
-    public static class IdentityServiceExtensions
+    public static class IdentityApplicationExtensions
     {
-        public static IIdentityServiceBuilder AddClientInfoBinding(this IIdentityServiceBuilder builder)
+        public static IIdentityServiceBuilder AddClientExtensions(this IIdentityServiceBuilder builder)
```

Comment:
I know we discussed Identity.Service => Identity.ClientApplication(s?), but maybe Identity.Client would mean roughly same thing, so would end up being mostly Application => Client which I think has the same semantics but is shorter